src/v/cloud_storage/tests/topic_manifest_test.cc

```diff
@@ -459,6 +459,7 @@ SEASTAR_THREAD_TEST_CASE(test_topic_manifest_serde_feature_table) {
       true,
       "rrr",
       std::nullopt,
+      std::nullopt,
```

Comment:
nit: commit message needs an update now that there is no archival_ntp

Reply:
Updated commit message.